app/src/components/Week.js

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+
+import dayjs from "dayjs";
+import CardGroup from "react-bootstrap/CardGroup";
+
+import DayOfWeek from "./DayOfWeek";
+
+const Week = () => {
+  const [startOfWeek, setStartOfWeek] = React.useState(
```

Comment:
why is the start of the week 1 day later instead of just ` dayjs().startOf("week")`?